### DISCUSSION TITLE
BTCPay 2.0 Translation - Report any missing string here

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#6295] [General]
Hey all,

We've done our best to cover most of the strings that can be translated, but it could happen that a string or two slipped through the cracks, if you identify a missing string (meaning that string appears in English after you've translated all into your local language) please let us know where it appears and we'll fix it in the next few minor releases as we make progress.

Thanks!

**Comment by pavlenex** [comment]
Here are some missing strings I discovered:

- [x] 1. **Create a store**  from a drop store selector
![Screenshot 2024-10-18 at 13 41 04](https://github.com/user-attachments/assets/dc6afec5-7d6e-4ff7-af69-7ac7c85081cf)
- [x] 2. **![Screenshot 2024-10-18 at 13 42 23](https://github.com/user-attachments/assets/27478108-175d-4cfa-9965-1ebe00ea0308)** on the dashboard
- [x] 3. Footer: **Donate, Copy Tor URL**
![Screenshot 2024-10-18 at 13 43 04](https://github.com/user-attachments/assets/930cc572-5973-4143-98b6-3a85247b798f)
- [x] 4. Server Settings > Policies, all headers: **Registration, User, Servers**, etc
![Screenshot 2024-10-18 at 13 43 52](https://github.com/user-attachments/assets/c96f914a-c7d0-41d7-97bc-881990420d3c)
- [x] 5. Nearly everything on Server Settings > Users
![Screenshot 2024-10-18 at 13 44 59](https://github.com/user-attachments/assets/11c5425c-c633-4582-a4a3-996b30f7aa81)
- [x] 6. Checkout experience sub-headers and drop downs for `less than`
![Uploading Screenshot 2024-10-18 at 13.47.
![Screenshot 2024-10-18 at 13 47 21](https://github.com/user-attachments/assets/4a324929-d77c-4042-95c0-ac7643c05b13)
12.png…]()
- [x] 7. API Keys entire page
![Screenshot 2024-10-18 at 13 48 06](https://github.com/user-attachments/assets/15064817-a0dc-4765-8db2-7f38a5929079)
- [x] 8. Store Users entier page
- [ ] 9. Roles a lot of stuff especially in tables 
![Screenshot 2024-10-18 at 13 48 51](https://github.com/user-attachments/assets/09007bf0-851c-4c3b-912b-c20c78a8762e)
- [x] 10. Webhooks descriptions
![Screenshot 2024-10-18 at 13 49 22](https://github.com/user-attachments/assets/b3a31320-2dcc-445c-8ede-cc5c9c59ecc0)
- [x] 11. Payout processors - nearly everything 
![Screenshot 2024-10-18 at 13 49 46](https://github.com/user-attachments/assets/82dad145-591e-4e86-8c85-9632908b7301)
- [x] 12. Email rules- everything 
![Screenshot 2024-10-18 at 13 50 19](https://github.com/user-attachments/assets/573359c1-6fe4-488e-9485-ac0dac167fc4)
- [x] 13. Forms - nearly everything 
![Screenshot 2024-10-18 at 13 50 51](https://github.com/user-attachments/assets/9c1faf6d-c5c4-4546-9219-0d82bbca743d)
- [ ] 14. Payment button nearly everything 
![Screenshot 2024-10-18 at 13 51 25](https://github.com/user-attachments/assets/3b3beeee-b710-4b1e-937d-0bc6c1635780)
- [x] 15. Plguin page nearly everything (we shouldn't translate plugin descriptions though)
![Screenshot 2024-10-18 at 13 52 05](https://github.com/user-attachments/assets/e27654a1-224c-4127-9d82-5d390eb02909)
- [x] 16. Invoices: headers and tables
![Screenshot 2024-10-18 at 13 52 58](https://github.com/user-attachments/assets/ab646e28-ea09-4c8a-b407-6bcc8f0de8fa)
![Screenshot 2024-10-18 at 13 53 02](https://github.com/user-attachments/assets/8e6e1aad-5bc8-451c-b1fb-f8fcf291c08c)
- [x] 17. Payouts
![Screenshot 2024-10-18 at 13 53 41](https://github.com/user-attachments/assets/88b993aa-880f-4bc4-bc2a-146a913f8de8)
 entire view

**Reply by dstrukt** [reply]
I created checkboxes out of these so we can step through them and confirm what's been done in full.

**Comment by dstrukt** [comment]
These are all (probably) minor, but very thoroughly documenting anyway:

# Supporters Section (Login, Signup, etc...)
- [ ] All Text
<img width="873" alt="Screen Shot 2024-10-21 at 4 49 58 PM" src="https://github.com/user-attachments/assets/6405262b-2377-406d-ae47-383955ce98b1">

----

# Account Settings

- [x] Account Settings 
<img width="997" alt="Screen Shot 2024-10-21 at 4 44 28 PM" src="https://github.com/user-attachments/assets/e4f1d21e-26a3-4260-b353-d70853668b2f">

- [x] Password Settings

<img width="1006" alt="Screen Shot 2024-10-21 at 4 47 02 PM" src="https://github.com/user-attachments/assets/3a3f0ba8-cebd-49ca-9c07-06ed80b768a8">

- [x] Access/2FA

<img width="935" alt="Screen Shot 2024-10-21 at 4 52 11 PM" src="https://github.com/user-attachments/assets/09cce2a4-0582-412c-8ee0-3de213b0466c">
<img width="507" alt="Screen Shot 2024-10-21 at 4 49 45 PM" src="https://github.com/user-attachments/assets/40653751-6c78-457f-89bd-07074ed6f1fe">

- [x] API Keys
<img width="763" alt="Screen Shot 2024-10-21 at 4 53 29 PM" src="https://github.com/user-attachments/assets/6899645b-2005-4c77-a0bf-317ca42e1be9">

- [x] Notification Settings

<img width="461" alt="Screen Shot 2024-10-21 at 4 54 07 PM" src="https://github.com/user-attachments/assets/3cf3b808-bd15-46a3-b6e7-7afd4aaa3617">

- [x] Login Codes

<img width="341" alt="Screen Shot 2024-10-21 at 4 54 31 PM" src="https://github.com/user-attachments/assets/6668d6cb-99a2-497d-86e3-93f69b3af759">

**Comment by dstrukt** [comment]
# Server Settings

- [x] Branding
<img width="993" alt="Screen Shot 2024-10-21 at 5 02 37 PM" src="https://github.com/user-attachments/assets/4e811108-700e-4ee7-b114-877c10fd1e64">

- [ ] Maintainence
<img width="1021" alt="Screen Shot 2024-10-21 at 5 01 18 PM" src="https://github.com/user-attachments/assets/d448de80-ce02-4713-b528-0f47a37c23fa">

- [ ] Logs
<img width="440" alt="Screen Shot 2024-10-21 at 5 06 41 PM" src="https://github.com/user-attachments/assets/f4fcaee7-ed9f-46d6-a2bb-ec70d817fdd7">

- [x] File Storage
<img width="1430" alt="Screen Shot 2024-10-21 at 5 07 10 PM" src="https://github.com/user-attachments/assets/837e1450-88b6-4a5d-aa94-318a50488492">

**Comment by dstrukt** [comment]
These are final catches on these views:

# Payment Requests

- [x] Search
<img width="985" alt="Screen Shot 2024-10-21 at 4 35 30 PM" src="https://github.com/user-attachments/assets/d6265844-1b05-4096-8d9e-c72c934c95f1">

# Invoices

- [x] Search
<img width="1004" alt="Screen Shot 2024-10-21 at 4 36 20 PM" src="https://github.com/user-attachments/assets/197f0694-45c9-4dd1-b716-2be0cc382841">

# Pull Payments
- [ ] Menu
<img width="562" alt="Screen Shot 2024-10-21 at 5 34 26 PM" src="https://github.com/user-attachments/assets/02bbcdcb-9c6e-4d12-a33b-5116c9f4efc4">

# Payouts
- [ ] Menu
<img width="740" alt="Screen Shot 2024-10-21 at 5 34 42 PM" src="https://github.com/user-attachments/assets/c405b2b6-5767-43a4-b21c-6d5eeb05d38b">

**Comment by dstrukt** [comment]
# Create Store

- [x] Should recommended be translated too?

<img width="457" alt="Screen Shot 2024-10-21 at 7 54 40 PM" src="https://github.com/user-attachments/assets/c3b525e5-74f2-4585-98d0-e32e7e9e4220">

# Store Settings

## Rates
- [x] This
<img width="953" alt="Screen Shot 2024-10-21 at 8 01 05 PM" src="https://github.com/user-attachments/assets/8ae6a523-1c5d-4f6d-bb4b-fe2462951a58">

- [x] Rate Modal
<img width="517" alt="Screen Shot 2024-10-21 at 7 59 58 PM" src="https://github.com/user-attachments/assets/7fc64744-c8da-4725-9dfa-a30a3fab3866">

## Users
- [x] This
<img width="1000" alt="Screen Shot 2024-10-21 at 8 01 52 PM" src="https://github.com/user-attachments/assets/c6f003d8-5cd9-41cb-8258-d686cf967754">

## Payout Processors
- [ ] This
<img width="979" alt="Screen Shot 2024-10-21 at 8 02 58 PM" src="https://github.com/user-attachments/assets/b4a807a5-b8c0-406c-9a90-08f841c89a20">

**Comment by dstrukt** [comment]
# Vault

- [ ] Done
<img width="750" alt="Screen Shot 2024-10-21 at 8 06 29 PM" src="https://github.com/user-attachments/assets/fc144349-9ece-4f6b-aec5-c417ed1ba90a">


# Wallet

- [ ] Done
<img width="665" alt="Screen Shot 2024-10-21 at 8 10 10 PM" src="https://github.com/user-attachments/assets/1b10e427-bc30-4ed2-9a42-7840ccb48ea0">

- [ ] Done
<img width="493" alt="Screen Shot 2024-10-21 at 8 11 01 PM" src="https://github.com/user-attachments/assets/163e79d1-7a36-41c8-b8db-b8b0f0aa3b84">

- [ ] Done
<img width="239" alt="Screen Shot 2024-10-21 at 8 12 08 PM" src="https://github.com/user-attachments/assets/cdcad37f-cdb3-4242-8a78-06197a8acd9f">

- [ ] Done
<img width="717" alt="Screen Shot 2024-10-21 at 8 12 42 PM" src="https://github.com/user-attachments/assets/06e82da6-b018-4f85-8e71-9e018cdfd6df">

- [ ] Done
<img width="283" alt="Screen Shot 2024-10-21 at 8 12 50 PM" src="https://github.com/user-attachments/assets/0735c67b-7fc7-4ccc-921a-a29addf08490">

# Lightning Node

- [ ] Done
<img width="428" alt="Screen Shot 2024-10-21 at 8 14 58 PM" src="https://github.com/user-attachments/assets/393414ad-1692-4ed4-9193-32433008a297">

- [ ] Done
<img width="404" alt="Screen Shot 2024-10-21 at 8 15 40 PM" src="https://github.com/user-attachments/assets/7b433257-dfda-4f3d-8a04-85060ad07f04">

**Comment by dstrukt** [comment]
# Dashboard
- [ ] Done
<img width="390" alt="Screen Shot 2024-10-21 at 8 19 16 PM" src="https://github.com/user-attachments/assets/8178eba7-92a5-42b2-be32-57bd3f50df6f">


# Point of Sale Settings

- [ ] Done
<img width="983" alt="Screen Shot 2024-10-21 at 8 19 29 PM" src="https://github.com/user-attachments/assets/48c05aff-26d1-4825-b9cf-8bfb6a602c7d">

- [ ] Done
<img width="723" alt="Screen Shot 2024-10-21 at 8 25 06 PM" src="https://github.com/user-attachments/assets/33103522-3be9-48cd-97c1-03c741a2c55a">

- [ ] Done
<img width="1014" alt="Screen Shot 2024-10-21 at 8 25 55 PM" src="https://github.com/user-attachments/assets/c03ab645-2db4-4fc1-af15-2f65140500c9">

- [ ] Done
<img width="414" alt="Screen Shot 2024-10-21 at 8 26 16 PM" src="https://github.com/user-attachments/assets/16934655-0b50-4bec-97d9-d555f5cbc70c">

- [ ] Done
<img width="975" alt="Screen Shot 2024-10-21 at 8 28 31 PM" src="https://github.com/user-attachments/assets/585580ef-73e6-4024-8381-f847aa4ebb5f">

- [ ] Done
<img width="958" alt="Screen Shot 2024-10-21 at 8 28 37 PM" src="https://github.com/user-attachments/assets/6b68b035-6f20-4302-9fa5-2de3c1bf10cb">

**Comment by TChukwuleta** [comment]
![image](https://github.com/user-attachments/assets/410480b7-23af-422e-931e-b95c2c45e274)

![image](https://github.com/user-attachments/assets/28dfbff5-e19f-47e2-bb37-b0b672216a32)

The Roles, payout processors, emails, forms tab too.
